advanced_html_css/animation/keyframes.md

```diff
@@ -115,7 +115,7 @@ Hopefully, this gives you a glimpse into the power the `@keyframes` syntax provi
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-1. Code along with the the MDN article for [using CSS animations](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations).
+1. Code along with the MDN article for [using CSS animations](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations).
```

Comment:
```suggestion
1. Code along with [this MDN article for using CSS animations](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations).
```

Thanks for fixing the typo. I also extended the description to become more descriptive if that's okay with you.

Reply:
yes, that's fine with me.

Reply:
will you click `commit suggestion` or add to your PR? 
You can leave the `resolve conversation` to the reviewers so they can check if the requested changes (if any) has satisfactorily been resolved if applicable :)

Reply:
I see! I'll `commit suggestion` now.